.github/workflows/bump-minor-version.yml

```diff
@@ -0,0 +1,122 @@
+          # NOTE: This workflow ignores conventional commit messages and always bumps the minor version.
+          default_bump: minor # Will go from v1.7.X -> v1.8.0
+          dry_run: true # Don't create tag yet, just calculate the new version
+
+      - name: 3. Set Variables for New Release
+        id: vars
+        run: |
+          # Use the 'new_tag' output, e.g., "v1.8.0"
+          NEW_V_VERSION="${{ steps.bump.outputs.new_tag }}"
+
+          # Extract major and minor version numbers
+          # Handles formats: v1.8.0, v1.8-dev, v1.8-rc1, etc.
+          MAJOR=$(echo "$NEW_V_VERSION" | sed -E 's/^v([0-9]+)\.([0-9]+).*/\1/')
+          MINOR=$(echo "$NEW_V_VERSION" | sed -E 's/^v([0-9]+)\.([0-9]+).*/\2/')
+
+          # Validate that we successfully extracted major and minor versions
+          if [[ -z "$MAJOR" ]] || [[ -z "$MINOR" ]]; then
+            echo "::error::Failed to extract major/minor version from: $NEW_V_VERSION"
+            exit 1
+          fi
+
+          # Validate that major and minor are numeric
+          if ! [[ "$MAJOR" =~ ^[0-9]+$ ]] || ! [[ "$MINOR" =~ ^[0-9]+$ ]]; then
+            echo "::error::Invalid version format. MAJOR=$MAJOR, MINOR=$MINOR (expected numeric values)"
+            exit 1
+          fi
+
+          # Create branch names
+          RELEASE_BRANCH="main-v${MAJOR}.${MINOR}"
+          WORK_BRANCH="gh-actions/bump-version-to-v${MAJOR}.${MINOR}"
+
+          echo "NEW_V_VERSION=${NEW_V_VERSION}" >> $GITHUB_OUTPUT
+          echo "RELEASE_BRANCH=${RELEASE_BRANCH}" >> $GITHUB_OUTPUT
+          echo "WORK_BRANCH=${WORK_BRANCH}" >> $GITHUB_OUTPUT
+
+          echo "::notice::Extracted versions - MAJOR: ${MAJOR}, MINOR: ${MINOR}"
+          echo "::notice::Creating release branch: ${RELEASE_BRANCH}"
+          echo "::notice::Creating work branch: ${WORK_BRANCH}"
+          echo "::notice::New version: ${NEW_V_VERSION}"
+
+      - name: 4. Create Release Branch
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+
+          RELEASE_BRANCH="${{ steps.vars.outputs.RELEASE_BRANCH }}"
+
+          # Create the release branch from main (use -B to reset if exists)
+          git checkout -B "$RELEASE_BRANCH"
+          git push origin "$RELEASE_BRANCH"
+
+          # The release branch is already checked out; no need to checkout again.
+
+
+          echo "::notice::Created and pushed release branch: $RELEASE_BRANCH"
+
+      - name: 5. Modify App Version in Code
+        run: |
+          VERSION_FILE="app/version/version.go"
+          NEW_VERSION="${{ steps.vars.outputs.NEW_V_VERSION }}" # e.g., "v1.8.0"
+
+          echo "::notice file=$VERSION_FILE,line=18::Bumping version in $VERSION_FILE to $NEW_VERSION"
+
+          # Use sed to find the line and replace only the version string
+          sed -i -E "s/^(var version = ).*/\1\"$NEW_VERSION\"/" "$VERSION_FILE"
```

Comment:
This will essentially update `v1.8-dev` to `v1.8.0`.
What we'd like so make it `v1.8-rc`, no?
The `v1.8.0` we should make with a `git tag` and change the version in this file to `v1.8`.